invenio_rdm_records/records/systemfields/statistics.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 TU Wien.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Cached transient field for record statistics."""
+
+from invenio_records.systemfields import SystemField
+from invenio_search.proxies import current_search_client
+from invenio_search.utils import build_alias_name
+
+from ...stats.utils import get_record_stats
+
+
+class RecordStatistics(SystemField):
```

Comment:
consistency with other sysfields:

```suggestion
class RecordStatisticsField(SystemField):
```